recipes/cppyythonizations/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "cppyythonizations" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/flatsurf/cppyythonizations/releases/download/{{ version }}/cppyythonizations-{{ version }}.tar.gz
+  sha256: 94cf064dc6c5f048b5987fb9dab295700403255ab993ff9b29cda9f3a9629517
+
+build:
+  noarch: python
```

This needs to compile code, so it can't be `noarch`. It looks like upstream doesn't support windows, so this should be instead:
```suggestion
  skip: True  # [win]
```

---

@dopplershift no, it's actually not compiling any code. It contains C++ code but it's only compiled on demand by cppyy on runtime. `cppyy` is a runtime dependency which is not packaged for windows yet. The `./configure` checks for the dependencies (because it can only run the test suite when they are present.) The package as such should work fine on Windows with the cppyy for Windows that can be obtained from PyPI.

---

Eventually, we'll make cppyy available for Windows on conda-forge. I just have not found the time yet to do it.

---

Fair enough (see below).